### DISCUSSION TITLE
Can't mark Intro to GitHub Flow as completed

### DISCUSSION THREAD

**Discussion by pabloverduzco** [community/community#123481] [GitHub Education]
Hi everyone,

I'm currently working on advancing through the Learning Path on GitHub Education, specifically on the "Intro to GitHub Flow" module. However, I've encountered an issue where I can't mark it as completed. The error message I'm receiving is: "**Rejected: We were unable to find a Classroom assignment for your account.**"

Has anyone else experienced this issue? I'd appreciate any insights or solutions you might have. Thank you in advance for your help!

<img width="1467" alt="Screenshot 2024-05-11 at 9 28 27 p m" src="https://github.com/community/community/assets/122489925/089387ad-af54-4d13-96f9-777e2f419bfc">
-
<img width="1467" alt="Screenshot 2024-05-11 at 9 30 04 p m" src="https://github.com/community/community/assets/122489925/223db194-9e21-4f2f-b51d-87c5dbf16b87">
-
<img width="1467" alt="Screenshot 2024-05-11 at 9 29 13 p m" src="https://github.com/community/community/assets/122489925/38221b11-fccb-4f94-8e2c-6f6694387e0c">

**Comment by CiceroLino** [comment]
Dumb question but... did you already have assigned to github education? To me, seems that github need to you assign github student developer pack

When you are assigned to github pack, you automatically have a github pro account and some others features of github student developer pack

**Reply by CiceroLino** [reply]
And also have this discussion, maybe this can help clarify something for you. 

https://github.com/orgs/community/discussions/111352

**Reply by pabloverduzco** [reply]
Hi CiceroLino,

Thanks for your response! That's a good point. Yes, I've already signed up for the GitHub Student Developer Pack and have a GitHub Pro account.

Do you think there might be something else causing the problem? I appreciate your help!

<img width="1467" alt="Screenshot 2024-05-11 at 9 46 02 p m" src="https://github.com/community/community/assets/122489925/d138ddf1-6331-49c0-81c1-3e7d4978a12e">

**Comment by kazmatw** [comment]
It sounds like you're encountering a fairly specific issue related to GitHub Education's integration with GitHub Classroom. Here are a few steps and checks you might consider to resolve the problem:

1. Verify Classroom Participation: Make sure that you are indeed part of a GitHub Classroom that is associated with the course you are trying to complete. Sometimes, you might not be properly linked or enrolled in the classroom that corresponds to the Learning Path module.

2. Check GitHub Account: Ensure that the GitHub account you are using is the one that has been invited or added to the Classroom. If you have multiple GitHub accounts, it's possible you might be logged into the wrong one.

3. Repository Setup: Verify that you have correctly set up any necessary repositories for the assignment. Some Learning Paths might require you to fork or clone a repository and push changes to it as part of the assignment.

4. Contact Course Administrator: There might be a setup issue on the course's end. Contacting the instructor or the course administrator might help clarify if there is an issue with the assignment setup or your inclusion in the course.

5. Refresh and Retry: Sometimes, simple technical glitches can be resolved by refreshing your browser or logging out and back into GitHub. It might help reset any session-specific issues that are preventing the completion from registering.

6. GitHub Support: If none of the above steps work, consider reaching out to GitHub Support. Provide them with details of the issue, any error messages you are seeing, and the steps you have already tried. They can provide more direct assistance, especially since the problem might be linked to specific configurations in GitHub Education or Classroom setups.

**Reply by pabloverduzco** [reply]
That's insightful, kazmatw. I believe you're onto something. It seems like my issue might be related to the GitHub Classroom integration. I've checked my Classroom participation, and it seems like there might be a problem with the assignment's link or its association with the class.

When I try to sync with the assignment invitation, I get redirected because it seems like my class doesn't exist. 

Do you have any suggestions on how I might handle this issue? Is there a way to re-link or re-enroll in the correct GitHub Classroom, or to resolve the problem with the assignment's link? Any guidance you can provide would be greatly appreciated!

<img width="1467" alt="Screenshot 2024-05-11 at 10 11 35 p m" src="https://github.com/community/community/assets/122489925/00c52d54-1a89-4539-af35-eebe212dbecd">
<img width="1467" alt="Screenshot 2024-05-11 at 10 11 59 p m" src="https://github.com/community/community/assets/122489925/6d6b9ff4-0ff9-4434-a581-aec7a32ed889">
<img width="1467" alt="Screenshot 2024-05-11 at 10 12 31 p m" src="https://github.com/community/community/assets/122489925/8272bf67-1fb8-456e-83a9-f4a4838ce928">

**Comment by CiceroLino** [comment]
Have you send a email to github staff? This problem seems more complicated what I thought

**Reply by pabloverduzco** [reply]
Thanks, CiceroLine, that's a good suggestion. I haven't reached out to GitHub staff yet, but it seems like this issue might indeed be a bit more complicated. I'll send them an email to see if they can provide any further assistance. Thanks for your input!

**Reply by CiceroLino** [reply]
Any news, I hope if you can share with us

**Reply by pabloverduzco** [reply]
Absolutely, CiceroLino. I've gone ahead and opened a ticket with GitHub support regarding the issue. Once I receive an update or a solution, I'll make sure to share it with everyone in this thread. Thank you for your interest and support!

**Comment by Brandon-Isaac** [comment]
# I am also experiencing this issue. What can I do about it?

**Comment by Brandon-Isaac** [comment]
You should try to move on to GitHub campus,, Surely you are done with the Introduction to github.
The achievement is that you know of it.
GitHub will update the GitHub education module

**Comment by MrChenLearnSpace** [comment]
Is it solved

**Reply by pabloverduzco** [reply]
Unfortunately, no. I decided to close the publication because it seems there is no a solution for this. I create multiple tickets and event the GitHub support team respond me back.
